lib/src/test/kotlin/starknet/utils/data/SnCastTypes.kt

```diff
@@ -57,7 +57,7 @@ data class AccountCreateSnCastResponse(
     val accountAddress: Felt,
 
     @JsonNames("max_fee")
-    val maxFee: Felt,
+    val maxFee: Long,
```

Comment:
`max_fee` is now returned as number, not as hex string.

Reply:
I think we can still represent it as hex string if we pass the `--hex-format` flag. However, this probably isn't necessary.